pkg/volume/azuredd/azure_common_windows.go

```diff
@@ -69,27 +69,27 @@ func findDiskByLun(lun int, iohandler ioHandler, exec utilexec.Interface) (strin
 				arr := strings.Split(location, " ")
 				arrLen := len(arr)
 				if arrLen < 3 {
-					klog.Warningf("unexpected json structure from Get-Disk, location: %q", jsonLocation)
+					klog.InfoS("Unexpected json structure from Get-Disk", "location", jsonLocation)
 					continue
 				}
 
-				klog.V(4).Infof("found a disk, location: %q, lun: %q", location, arr[arrLen-1])
+				klog.V(4).InfoS("Found a disk", "location", location, "lun", arr[arrLen-1])
 				//last element of location field is LUN number, e.g.
 				//		"location":  "Integrated : Adapter 3 : Port 0 : Target 0 : LUN 1"
 				l, err := strconv.Atoi(arr[arrLen-1])
 				if err != nil {
-					klog.Warningf("cannot parse element from data structure, location: %q, element: %q", location, arr[arrLen-1])
+					klog.InfoS("Cannot parse element from data structure", "location", location, "element", arr[arrLen-1])
 					continue
 				}
 
 				if l == lun {
-					klog.V(4).Infof("found a disk and lun, location: %q, lun: %d", location, lun)
+					klog.V(4).InfoS("Found a disk and lun", "location", location, "lun", lun)
```

Comment:
What is `lun` here?

Reply:
Hi, @serathius. `lun` is the shortname for `Logical Unit Number` in Storage Area Network (SAN).

Reply:
Let's use full name in logs Logical Unit Number and capitalized key LUN as this is an acronym.

Reply:
Hi, @serathius. We have updated `lun` in logs with `Logical Unit Number`, and replaced key `lun` with `LUN` at the same time.